src/js/popup.js

```diff
@@ -1,8 +1,11 @@
-import "../css/popup.css";
+import '../css/popup.css';
 import '../css/bootstrap.min.css';
 
 import { createStore, applyMiddleware } from 'redux';
 import reduxThunk from 'redux-thunk';
+import { faPlayCircle, faStopCircle } from '@fortawesome/free-regular-svg-icons';
+import { library, dom } from '@fortawesome/fontawesome-svg-core';
```

Comment:
Can we rename it from `library` to something else?

In code if you see library it's not clear what library it is.